.github/ISSUE_TEMPLATE/bug_report_template.yaml

```diff
@@ -0,0 +1,153 @@
+name: Bug Report
+description: Create a report to help us reproduce and correct the bug
+labels: ['Bug', 'Needs Triage']
+title: '[Bug]'
+
+body:
+  - type: markdown
+    attributes:
+      value: | 
+        Thank you for submitting an issue. Please refer to our [issue policy](https://www.github.com/mlflow/mlflow/blob/master/ISSUE_POLICY.md) for additional information about bug reports. For help with debugging your code, please refer to [Stack Overflow](https://stackoverflow.com/questions/tagged/mlflow).
+        #### Please fill in this bug report template to ensure a timely and thorough response.
+  - type: checkboxes
+    id: contribution
+    attributes:
+      label: Willingness to contribute
+      description: The MLflow Community encourages bug fix contributions. Would you or another member of your organization be willing to contribute a fix for this bug to the MLflow code base?
+      options:
+        - label: Yes. I can contribute a fix for this bug independently.
+          required: false
+        - label: Yes. I would be willing to contribute a fix for this bug with guidance from the MLflow community.
+          required: false
+        - label: No. I cannot contribute a bug fix at this time.
+          required: false
+  - type: markdown
+    attributes:
+      value: | 
+        ### System information
+        #### Please fill in this bug report template to ensure a timely and thorough response.
+        - **Have I written custom code (as opposed to using a stock example script provided in MLflow)**:
+        - **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
+        - **MLflow installed from (source or binary)**:
+        - **MLflow version (run ``mlflow --version``)**:
+        - **Python version**:
+        - **npm version, if running the dev UI**:
+        - **Exact command to reproduce**:
```

Comment:
Can we use a textarea or input boxes for this section?

Reply:
We might not need `**Exact command to reproduce**:` because we have the Code-to-reproduce-issue section